pytorch_lightning/trainer/trainer.py

```diff
@@ -467,27 +475,29 @@ def fit(
         self.accelerator_backend.setup(self, model)
         self.setup_trainer(model)
 
+        # ----------------------------
+        # INSPECT THESE FOR MAIN LOOPS
+        # ----------------------------
+        # assign training and eval functions... inspect these to see the train and eval loops :)
+        self.accelerator_backend.train_loop = self.run_train
+        self.accelerator_backend.validation_loop = self.run_evaluation
+        self.accelerator_backend.test_loop = self.run_evaluation
+        self.accelerator_backend.predict_loop = self.run_predict
+
```

Comment:
@tchaton I think it's a merge problem, it shouldn't be here. This is old.

Reply:
Actually, it should be there and I don't understand why it was removed.

Reply:
It is absolutely useless code-wise :) But Will wanted this to help users understand where the loops are. It shouldn't be removed.